package/InternalEAW.roc

```diff
@@ -0,0 +1,10 @@
+## WARNING This file is automatically generated. Do not edit it manually. ##
+module [eastAsianWidthProperty, eastAsianWidth]
+
+eastAsianWidth : U32 -> U32
```

Comment:
Added a type annotation here so that the compiler will only generate one variant.